Backend/app/dashboard/Lib/src/API/Dashboard/Person.hs

```diff
@@ -80,6 +95,9 @@ handler =
       :<|> assignMerchantAccess
       :<|> resetMerchantAccess
       :<|> createPerson
+      :<|> changeEmailByAdmin
```

Comment:
No need to mention ByAdmin

Reply:
@hkmangla , their is already a function change password , so have to make name different so made it byAdmin and keep it similar for email and mobile also.